aten/src/ATen/core/boxing/KernelFunction.cpp

```diff
@@ -16,4 +18,27 @@ void fallthrough_kernel(OperatorKernel*, const OperatorHandle&, Stack*) {
     "let us know in the bug tracker.");
 }
 
+// single line summary of state
+std::string KernelFunction::dumpState() const {
```

Comment:
I feel like the likelihood of misuse is low (since the names are reasonably clear), and I actually *want* people to use these methods outside of test cases, e.g., when they are debugging. I definitely don't want them to be private; if you're debugging some problem and you want to stick a printf debug, I want to let you do that. If you like, I could add leading underscores to them, how does that sound?